pom.xml

```diff
@@ -78,6 +69,33 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>${java.version}</source>
+                    <target>${java.version}</target>
+                    <annotationProcessorPaths>
```

Comment:
Configure annotation processors in maven. Copy/pasted shamelessly from https://github.com/springframeworkguru/spring5-mvc-rest/blob/mapstruct/pom.xml#L89